index.js

```diff
@@ -83,6 +67,27 @@ const slowDownPlugin = async (fastify, settings) => {
       return reply
     }
   })
+
+  fastify.addHook('onSend', async (req, reply) => {
+    const key = options.keyGenerator(req)
+    if (options.skipFailedRequests) {
+      if (reply.statusCode >= 400) {
```

Comment:
how about we merge these two conditions in a single if? :)

Reply:
+1